website_sale_credit_point/templates/navbar_points.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <template id="navbar_points" inherit_id="portal.frontend_layout">
+        <xpath expr="//ul[@id='top_menu']/*[1]" position="before">
+            <li t-if="user_id.partner_id">
+                <!-- i use empty a tag here to get the styles, used in top menu -->
```

Comment:
"empty a tag"? Perhaps you mean "an empty tag"?

Reply:
No, that's not what I meant, but I will rephrase it to be more clear :D